Project.toml

```
@@ -13,7 +13,7 @@ Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-AbstractMCMC = "2, 3, 4"
+AbstractMCMC = "2, 3"
 DensityInterface = "0.4"
-Setfield = "0.7.1, 0.8"
-julia = "~1.6.6, 1.7.3"
+Setfield = "0.8.1"
+julia = "1.6"
```

This reenables support for Julia 1.6.0-1.6.5 it seems? Is this desired?
```suggestion
julia = "~1.6.6, 1.7.3"
```